src/_mixins/use-is-composition.ts

```diff
@@ -0,0 +1,63 @@
+import { ref, Ref, onMounted, onUnmounted } from 'vue'
+
+export const useIsComposition: () => {
```

Comment:
onIsCompositing

https://www.wordhippo.com/what-is/the-verb-for/composition.html